From b53f81f216513824eca9df8e0594a1c995520030 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Sun, 17 Feb 2013 23:39:08 +0100
Subject: Use makeinfo instead of texi2html

Texi2html is no longer maintained so this commit convert the Makefile to
use makeinfo instead of texi2html.

Origin: upstream, http://repo.or.cz/w/tinycc.git/commit/1d673cbfd619995f2762d3e216f8f0f842effc8c
Forwarded: http://repo.or.cz/w/tinycc.git/commit/1d673cbfd619995f2762d3e216f8f0f842effc8c
Last-Update: 2013-05-29
Applied-Upstream: commit: 1d673cbfd619995f2762d3e216f8f0f842effc8c
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b755589..4049d8a 100644
--- a/Makefile
+++ b/Makefile
@@ -300,7 +300,7 @@ endif
 
 # documentation and man page
 tcc-doc.html: tcc-doc.texi
-	-texi2html -monolithic -number $<
+	-makeinfo --no-split --html --number-sections -o $@ $<
 
 tcc.1: tcc-doc.texi
 	-$(top_srcdir)/texi2pod.pl $< tcc.pod
